### PR TITLE
fixes infinite creampie stacking

### PR DESCRIPTION
**Pull request by silicons**
NO FUN ALLOWED
:cl:
bugfix: You can no longer have mutliple creampies on your face.
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks_pie.dm [hunk 1]
```diff
@@ -41,7 +41,8 @@
 			H.Weaken(1) //splat!
 			H.adjust_blurriness(1)
 			visible_message("<span class='userdanger'>[H] was creamed by [src]!!</span>")
-			H.add_overlay(creamoverlay)
+			if(!(creamoverlay in H.overlays))
+				H.add_overlay(creamoverlay)
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/food/snacks_pie.dm]: This is not the way to do this, absolutely nothing aside from SSoverlays should be using the overlays var. Make a different var instead

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by xxalpha** [comment]
Change title to "breaks infinite creampie stacking".

**Comment by kevinz000** [comment]
can't be assed to do this

**Comment by FantasticFwoosh** [comment]
Congratulations you backed out of your fix for a weapon that'll be removed because of infinite creampie stacking.

**Comment by Ausops** [comment]
someone was getting multiple creampies on their face huh

**Comment by Cyberboss** [comment]
FIX IT YOU NERD

**Comment by Anonmare** [comment]
>Local coder gets tabled and creampied over and over: Aheago edition

**Comment by xxalpha** [comment]
Sounds like rape

**Pull request closed without merging**